apple-nav/src/App.js

```diff
@@ -0,0 +1,109 @@
+import React, { Component } from 'react';
+import Data from './data';
+import styled, { createGlobalStyle } from '../node_modules/styled-components';
+import PropTypes from 'prop-types';
+
+import NavBar from './components/NavBar';
+
+const GlobalStyle = createGlobalStyle`
```

Comment:
Good job incorporating styled-components. I really didn't like them at first and stopped using them after last week, but I've since started using them more and more and I'm a big fan now. They are really powerful, I just didn't understand them. I'm glad your using them.